docs/oscar_references.bib

```diff
@@ -1365,3 +1365,13 @@ @Misc{cohomCalg:Implementation
   year          = {2010},
   url           = {https://github.com/BenjaminJurke/cohomCalg}
 }
+
+@Article{EHU03,
```

Comment:
This Bib-Entry is missing the publication year. Didn't you already resolve this Problem in the preceding PR ProjectiveBundlesII ?
(Is this indicating that the PR still needs to be made up-to-date?)

Reply:
The diff is messed up as well. Rebase on Master?